plugins/TimingFanoutController.cpp

```diff
@@ -48,7 +48,7 @@ TimingFanoutController::TimingFanoutController(const std::string& name)
 void
 TimingFanoutController::do_configure(const nlohmann::json& data)
 {
-  auto mdal = m_params->module<dal::TimingFanoutControllerParameters>(get_name()); 
+  auto mdal = m_params->cast<dal::TimingFanoutController>(); 
```

Comment:
out of curiosity, why are you using `cast` instead of `module`?

Reply:
Because of the restructuring of the schema, TimingFanoutController inherits from TimingController. m_params is now a dal::TimingController*, so the cast changes it to dal::TimingFanoutController*